frontend-cert/html-projects/travel-agency-page/user-stories.md

```diff
@@ -0,0 +1,15 @@
+1. You should have an `h1` element with the text `Discover Italy`.
+
+2. You should have a paragraph below the `h1` introducing the travel opportunities.
+
+3. You should have an `h2` element with the text `Packages`.
+
+4. You should have a `p` element introducing briefly the various packages.
+
+5. You should have an unordered list element with two list items. The two list items should have the text `Group Travels` and `Private Tours`, respectively. The text of each list item should be enclosed by an anchor element with a dead `href` attribute as placeholder, and a `target` attribute of `_blank`.
+
+6. You should have an `h2` element with the text `Top Itineraries`.
+
+7. You should have three `figure elements`, each containing an `img` element inside an anchor element, and a `figcaption`.
+
+8. For each `figure` element, you should use respectively as image `src` and `figcatpion` text: `colosseo.jpg` and `Rome and Center Italy`; `lavaredo.jpg` and `National Parks`; and `favignana.jpg` and `South Italy and Islands`.
```

Comment:
This feels like it could be a little confusing to some. What about...

```suggestion
7. You should have three `figure` elements, each containing an anchor element and a `figcaption` element. The three anchor elements should have an `img` element as its content.

8. For each `figure` element, use the following image `src` and `figcaption` values respectively: `colosseo.jpg` with `Rome and Center Italy`, `lavaredo.jpg` with `National Parks`, `favignana.jpg` with `South Italy and Islands`.
```

It doesn't really mention what to do with the anchor elements - which is fine. But we could add href and _blank as a requirement? Or perhaps a single user story somewhere that says "All your anchor elements should have href and _blank". Then we could delete the earlier part about those.